docs/linux_dependencies.md

```diff
@@ -137,7 +137,9 @@ Note that this template does not add Rust to the environment because there are m
    sudo emerge --ask libX11 pkgconf alsa-lib
 ```
 
-When using an AMD Radeon GPU, you may also need to emerge `amdgpu-pro-vulkan` to get Bevy to find the GPU.
+When using an AMD Radeon GPU, you may also need to emerge `amdgpu-pro-vulkan` to get Bevy to find the GPU. 
+
+When using a NVIDIA GPU with the proprietary driver (eg. `x11-drivers/nvidia-drivers`), you may also need to emerge `media-libs/vulkan-loader` to get Bevy to find the GPU. NVIDIA Vulkan driver is included in `nvidia-driver`, but may need the loader to find the correct ICD. See Gentoo [Documentation](https://wiki.gentoo.org/wiki/) for details..
```

Comment:
Is there a more specific link to the docs you can give?

Can you expand the ICD acronym? It's giving me trouble to Google and I'm not familiar with it.

Reply:
My Bad, pasted the wrong link. Fixed it.

Reply:
`ICD` is the acronym used in the Gentoo doc, I guess `driver` will suffice.